src/me/drton/flightplot/FlightPlot.java

```diff
@@ -729,6 +729,11 @@ public void showImportPresetDialog() {
                 Preset preset = Preset.unpackJSONObject(new JSONObject(new String(b, Charset.forName("utf8"))));
                 loadPreset(preset);
                 processFile();
+                if (preset.getTitle().equals("")) {
+                    // generate a namestring
+                    preset.setTitle(file.getName().substring(0, file.getName().indexOf(".fplot")));
+                }
+                presetComboBox.addItem(preset);
```

Comment:
Although I can see the practicality '). At least, the imported preset should become the selected item in the list if added.